orchestra/static/orchestra/common/js/orchestra_api.js

```diff
@@ -0,0 +1,62 @@
+var serviceModule =  angular.module('orchestra.common.services');
+
+serviceModule.factory('orchestraApi', ['$http', function($http) {
+  var actionService = new function() {
+    var actionService = this;
+
+    actionService.projectManagementData = function(projectId) {
+      return $http.post('/orchestra/api/interface/project_management_information/', {
```

Comment:
minor nitpick: consider avoiding repeating `/orchestra/api/interface/` in case we change that URL---have it in one variable?

OK if you don't change :)